recipes/restore_sql.rb

```diff
@@ -4,14 +4,15 @@
 mgm_connection = node['ndb']['connectstring']
 should_run = private_ip.eql?(node['ndb']['mysqld']['private_ips'].sort[0])
 mysql_cli = "#{node['ndb']['scripts_dir']}/mysql-client.sh"
+exclude_databases=node['ndb']['restore']['exclude_databases_meta']
 rebuild_indexes_check = "#{Chef::Config['file_cache_path']}/rondb_rebuild_indexes_#{node['install']['version']}_#{node['ndb']['version']}"
 
 bash 'Rebuild indexes' do
     user 'root'
     group 'root'
     code <<-EOH
         set -e
-        #{node['ndb']['scripts_dir']}/restore_backup.sh ndb-restore -p #{backup_directory} -n 1 -b #{node['ndb']['restore']['backup_id']} -c #{mgm_connection} -m REBUILD-INDEXES
+        #{node['ndb']['scripts_dir']}/restore_backup.sh ndb-restore -p #{backup_directory} -n 1 -b #{node['ndb']['restore']['backup_id']} -c #{mgm_connection} -e #{exclude_databases} -m REBUILD-INDEXES
```

Comment:
Why is REBUILD-INDEXES in capital letters

Reply:
It's an argument to the wrapper script, it's not passed directly to ndb_restore